crates/ironrdp-displaycontrol/src/pdu/mod.rs

```diff
@@ -30,6 +31,18 @@ pub enum DisplayControlPdu {
 impl DisplayControlPdu {
     const NAME: &'static str = "DISPLAYCONTROL_HEADER";
     const FIXED_PART_SIZE: usize = 4 /* Type */ + 4 /* Length */;
+
+    /// Creates a new [`DisplayControlPdu::MonitorLayout`] PDU for a single monitor with the given width and height.
+    pub fn create_monitor_layout_pdu(width: u32, height: u32) -> PduResult<Self> {
```

Comment:
Lets move this to `DisplayControlMonitorLayout` as an alternative constructor and name it something like `pub new_single_monitor(width: u32, height: u32)`

Reply:
50c649e5f4f993979948fcad5f6b7b436b79ea7a